### PR TITLE
fix(migration): 0008 needs to be backported to stable so use it as 0011

### PR DESCRIPTION
**Pull request by william-gr**

### CODE REVIEW

**Comment by b-a-t** [comment]
rebuild

File: gui/sharing/migrations/0011_aio_pthread_remove.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            filter(lambda v: v not in {'aio_pthread'}, share.cifs_vfsobjects)
+        )
+        share.cifs_vfsobjects = new_vfs_objects
+        share.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('sharing', '0007_afp_share_afp_home'),
```
Comment (themylogin) [inline gui/sharing/migrations/0011_aio_pthread_remove.py]: should be `0009_afp_share_afp_timemachine_quota_in_gib_instead_of_mib`

Comment (william-gr) [reply gui/sharing/migrations/0011_aio_pthread_remove.py]: But why? Its a exact copy of 0008 and it worked, I thought they were idempotent.

Comment (themylogin) [reply gui/sharing/migrations/0011_aio_pthread_remove.py]: @william-gr because we don't want do have `0007_afp_share_afp_home` in stable

Comment (william-gr) [reply gui/sharing/migrations/0011_aio_pthread_remove.py]: @themylogin I dont get it. What is the problem with that? In stable it references 0009. Is it simply to keep them the same between branches?

Comment (william-gr) [reply gui/sharing/migrations/0011_aio_pthread_remove.py]: @themylogin right I but thought you meant the migration code, not the metadata. I will change tomorrow.

**Comment by kmoore134** [comment]
rebuild

**Comment by william-gr** [comment]
@themylogin I believe I have addressed your concern and will merge it now. If not let me know, thanks.

**Pull request merged**